README.md

```diff
@@ -66,6 +67,33 @@ Useful Helm Client Commands:
 - Install a chart: `helm install my-release oci://registry-1.docker.io/bitnamicharts/<chart>`
 - Upgrade your application: `helm upgrade my-release oci://registry-1.docker.io/bitnamicharts/<chart>`
 
+## Creating a new chart
+
+To make a chart that follows the same structure/patterns that the rest of the Bitnami charts, the basic scaffolding is provided in the [`template` directory](https://github.com/bitnami/charts/tree/main/template).
```

Comment:
Why don't you use a relative reference for `./template` as you are doing for other links? Does it make sense?

Reply:
I copied the link from CONTRIBUTING.md, so kept it as it was. I later updated the linter to allow relative links. I have now made this link relative too.